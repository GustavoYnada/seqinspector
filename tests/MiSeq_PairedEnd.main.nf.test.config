// Load the basic test config
includeConfig 'nextflow.config'

// Load the correct samplesheet for that test
params {
    input  = params.pipelines_testdata_base_path + 'seqinspector/testdata/Miseq_PairedEnd/samplesheet.csv'
}
