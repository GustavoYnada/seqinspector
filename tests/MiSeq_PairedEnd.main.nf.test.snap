{
    "MiSeq data test": {
        "content": [
            8,
            {
                "FASTQC": {
                    "fastqc": "0.12.1"
                },
                "FASTQSCREEN_FASTQSCREEN": {
                    "fastqscreen": "0.16.0"
                },
                "SEQFU_STATS": {
                    "seqfu": "1.20.3"
                },
                "Workflow": {
                    "nf-core/seqinspector": "v1.0dev"
                }
            },
            [
                "fastqc",
                "fastqc/SAMPLE_PAIRED_END_1_1_1_fastqc.html",
                "fastqc/SAMPLE_PAIRED_END_1_1_1_fastqc.zip",
                "fastqc/SAMPLE_PAIRED_END_1_1_2_fastqc.html",
                "fastqc/SAMPLE_PAIRED_END_1_1_2_fastqc.zip",
                "fastqc/SAMPLE_PAIRED_END_2_2_1_fastqc.html",
                "fastqc/SAMPLE_PAIRED_END_2_2_1_fastqc.zip",
                "fastqc/SAMPLE_PAIRED_END_2_2_2_fastqc.html",
                "fastqc/SAMPLE_PAIRED_END_2_2_2_fastqc.zip",
                "fastqscreen",
                "fastqscreen/SAMPLE_PAIRED_END_1_1_1_screen.html",
                "fastqscreen/SAMPLE_PAIRED_END_1_1_1_screen.png",
                "fastqscreen/SAMPLE_PAIRED_END_1_1_1_screen.txt",
                "fastqscreen/SAMPLE_PAIRED_END_1_1_2_screen.html",
                "fastqscreen/SAMPLE_PAIRED_END_1_1_2_screen.png",
                "fastqscreen/SAMPLE_PAIRED_END_1_1_2_screen.txt",
                "fastqscreen/SAMPLE_PAIRED_END_2_2_1_screen.html",
                "fastqscreen/SAMPLE_PAIRED_END_2_2_1_screen.png",
                "fastqscreen/SAMPLE_PAIRED_END_2_2_1_screen.txt",
                "fastqscreen/SAMPLE_PAIRED_END_2_2_2_screen.html",
                "fastqscreen/SAMPLE_PAIRED_END_2_2_2_screen.png",
                "fastqscreen/SAMPLE_PAIRED_END_2_2_2_screen.txt",
                "multiqc",
                "multiqc/global_report",
                "multiqc/global_report/multiqc_data",
                "multiqc/global_report/multiqc_data/fastq_screen_plot.txt",
                "multiqc/global_report/multiqc_data/fastqc-status-check-heatmap.txt",
                "multiqc/global_report/multiqc_data/fastqc_adapter_content_plot.txt",
                "multiqc/global_report/multiqc_data/fastqc_per_base_n_content_plot.txt",
                "multiqc/global_report/multiqc_data/fastqc_per_base_sequence_quality_plot.txt",
                "multiqc/global_report/multiqc_data/fastqc_per_sequence_gc_content_plot_Counts.txt",
                "multiqc/global_report/multiqc_data/fastqc_per_sequence_gc_content_plot_Percentages.txt",
                "multiqc/global_report/multiqc_data/fastqc_per_sequence_quality_scores_plot.txt",
                "multiqc/global_report/multiqc_data/fastqc_sequence_counts_plot.txt",
                "multiqc/global_report/multiqc_data/fastqc_sequence_duplication_levels_plot.txt",
                "multiqc/global_report/multiqc_data/fastqc_top_overrepresented_sequences_table.txt",
                "multiqc/global_report/multiqc_data/multiqc.log",
                "multiqc/global_report/multiqc_data/multiqc_SAMPLE_PAIRED_END_1_1.txt",
                "multiqc/global_report/multiqc_data/multiqc_SAMPLE_PAIRED_END_2_2.txt",
                "multiqc/global_report/multiqc_data/multiqc_citations.txt",
                "multiqc/global_report/multiqc_data/multiqc_data.json",
                "multiqc/global_report/multiqc_data/multiqc_fastq_screen.txt",
                "multiqc/global_report/multiqc_data/multiqc_fastqc.txt",
                "multiqc/global_report/multiqc_data/multiqc_general_stats.txt",
                "multiqc/global_report/multiqc_data/multiqc_software_versions.txt",
                "multiqc/global_report/multiqc_data/multiqc_sources.txt",
                "multiqc/global_report/multiqc_plots",
                "multiqc/global_report/multiqc_plots/pdf",
                "multiqc/global_report/multiqc_plots/pdf/SAMPLE_PAIRED_END_1_1.pdf",
                "multiqc/global_report/multiqc_plots/pdf/SAMPLE_PAIRED_END_2_2.pdf",
                "multiqc/global_report/multiqc_plots/pdf/fastq_screen_plot-cnt.pdf",
                "multiqc/global_report/multiqc_plots/pdf/fastq_screen_plot-pct.pdf",
                "multiqc/global_report/multiqc_plots/pdf/fastqc-status-check-heatmap.pdf",
                "multiqc/global_report/multiqc_plots/pdf/fastqc_adapter_content_plot.pdf",
                "multiqc/global_report/multiqc_plots/pdf/fastqc_per_base_n_content_plot.pdf",
                "multiqc/global_report/multiqc_plots/pdf/fastqc_per_base_sequence_quality_plot.pdf",
                "multiqc/global_report/multiqc_plots/pdf/fastqc_per_sequence_gc_content_plot_Counts.pdf",
                "multiqc/global_report/multiqc_plots/pdf/fastqc_per_sequence_gc_content_plot_Percentages.pdf",
                "multiqc/global_report/multiqc_plots/pdf/fastqc_per_sequence_quality_scores_plot.pdf",
                "multiqc/global_report/multiqc_plots/pdf/fastqc_sequence_counts_plot-cnt.pdf",
                "multiqc/global_report/multiqc_plots/pdf/fastqc_sequence_counts_plot-pct.pdf",
                "multiqc/global_report/multiqc_plots/pdf/fastqc_sequence_duplication_levels_plot.pdf",
                "multiqc/global_report/multiqc_plots/pdf/fastqc_top_overrepresented_sequences_table.pdf",
                "multiqc/global_report/multiqc_plots/pdf/general_stats_table.pdf",
                "multiqc/global_report/multiqc_plots/png",
                "multiqc/global_report/multiqc_plots/png/SAMPLE_PAIRED_END_1_1.png",
                "multiqc/global_report/multiqc_plots/png/SAMPLE_PAIRED_END_2_2.png",
                "multiqc/global_report/multiqc_plots/png/fastq_screen_plot-cnt.png",
                "multiqc/global_report/multiqc_plots/png/fastq_screen_plot-pct.png",
                "multiqc/global_report/multiqc_plots/png/fastqc-status-check-heatmap.png",
                "multiqc/global_report/multiqc_plots/png/fastqc_adapter_content_plot.png",
                "multiqc/global_report/multiqc_plots/png/fastqc_per_base_n_content_plot.png",
                "multiqc/global_report/multiqc_plots/png/fastqc_per_base_sequence_quality_plot.png",
                "multiqc/global_report/multiqc_plots/png/fastqc_per_sequence_gc_content_plot_Counts.png",
                "multiqc/global_report/multiqc_plots/png/fastqc_per_sequence_gc_content_plot_Percentages.png",
                "multiqc/global_report/multiqc_plots/png/fastqc_per_sequence_quality_scores_plot.png",
                "multiqc/global_report/multiqc_plots/png/fastqc_sequence_counts_plot-cnt.png",
                "multiqc/global_report/multiqc_plots/png/fastqc_sequence_counts_plot-pct.png",
                "multiqc/global_report/multiqc_plots/png/fastqc_sequence_duplication_levels_plot.png",
                "multiqc/global_report/multiqc_plots/png/fastqc_top_overrepresented_sequences_table.png",
                "multiqc/global_report/multiqc_plots/png/general_stats_table.png",
                "multiqc/global_report/multiqc_plots/svg",
                "multiqc/global_report/multiqc_plots/svg/SAMPLE_PAIRED_END_1_1.svg",
                "multiqc/global_report/multiqc_plots/svg/SAMPLE_PAIRED_END_2_2.svg",
                "multiqc/global_report/multiqc_plots/svg/fastq_screen_plot-cnt.svg",
                "multiqc/global_report/multiqc_plots/svg/fastq_screen_plot-pct.svg",
                "multiqc/global_report/multiqc_plots/svg/fastqc-status-check-heatmap.svg",
                "multiqc/global_report/multiqc_plots/svg/fastqc_adapter_content_plot.svg",
                "multiqc/global_report/multiqc_plots/svg/fastqc_per_base_n_content_plot.svg",
                "multiqc/global_report/multiqc_plots/svg/fastqc_per_base_sequence_quality_plot.svg",
                "multiqc/global_report/multiqc_plots/svg/fastqc_per_sequence_gc_content_plot_Counts.svg",
                "multiqc/global_report/multiqc_plots/svg/fastqc_per_sequence_gc_content_plot_Percentages.svg",
                "multiqc/global_report/multiqc_plots/svg/fastqc_per_sequence_quality_scores_plot.svg",
                "multiqc/global_report/multiqc_plots/svg/fastqc_sequence_counts_plot-cnt.svg",
                "multiqc/global_report/multiqc_plots/svg/fastqc_sequence_counts_plot-pct.svg",
                "multiqc/global_report/multiqc_plots/svg/fastqc_sequence_duplication_levels_plot.svg",
                "multiqc/global_report/multiqc_plots/svg/fastqc_top_overrepresented_sequences_table.svg",
                "multiqc/global_report/multiqc_plots/svg/general_stats_table.svg",
                "multiqc/global_report/multiqc_report.html",
                "multiqc/group_reports",
                "multiqc/group_reports/Bpacificus",
                "multiqc/group_reports/Bpacificus/multiqc_data",
                "multiqc/group_reports/Bpacificus/multiqc_data/fastq_screen_plot.txt",
                "multiqc/group_reports/Bpacificus/multiqc_data/fastqc-status-check-heatmap.txt",
                "multiqc/group_reports/Bpacificus/multiqc_data/fastqc_adapter_content_plot.txt",
                "multiqc/group_reports/Bpacificus/multiqc_data/fastqc_per_base_n_content_plot.txt",
                "multiqc/group_reports/Bpacificus/multiqc_data/fastqc_per_base_sequence_quality_plot.txt",
                "multiqc/group_reports/Bpacificus/multiqc_data/fastqc_per_sequence_gc_content_plot_Counts.txt",
                "multiqc/group_reports/Bpacificus/multiqc_data/fastqc_per_sequence_gc_content_plot_Percentages.txt",
                "multiqc/group_reports/Bpacificus/multiqc_data/fastqc_per_sequence_quality_scores_plot.txt",
                "multiqc/group_reports/Bpacificus/multiqc_data/fastqc_sequence_counts_plot.txt",
                "multiqc/group_reports/Bpacificus/multiqc_data/fastqc_sequence_duplication_levels_plot.txt",
                "multiqc/group_reports/Bpacificus/multiqc_data/fastqc_top_overrepresented_sequences_table.txt",
                "multiqc/group_reports/Bpacificus/multiqc_data/multiqc.log",
                "multiqc/group_reports/Bpacificus/multiqc_data/multiqc_SAMPLE_PAIRED_END_1_1.txt",
                "multiqc/group_reports/Bpacificus/multiqc_data/multiqc_SAMPLE_PAIRED_END_2_2.txt",
                "multiqc/group_reports/Bpacificus/multiqc_data/multiqc_citations.txt",
                "multiqc/group_reports/Bpacificus/multiqc_data/multiqc_data.json",
                "multiqc/group_reports/Bpacificus/multiqc_data/multiqc_fastq_screen.txt",
                "multiqc/group_reports/Bpacificus/multiqc_data/multiqc_fastqc.txt",
                "multiqc/group_reports/Bpacificus/multiqc_data/multiqc_general_stats.txt",
                "multiqc/group_reports/Bpacificus/multiqc_data/multiqc_software_versions.txt",
                "multiqc/group_reports/Bpacificus/multiqc_data/multiqc_sources.txt",
                "multiqc/group_reports/Bpacificus/multiqc_plots",
                "multiqc/group_reports/Bpacificus/multiqc_plots/pdf",
                "multiqc/group_reports/Bpacificus/multiqc_plots/pdf/SAMPLE_PAIRED_END_1_1.pdf",
                "multiqc/group_reports/Bpacificus/multiqc_plots/pdf/SAMPLE_PAIRED_END_2_2.pdf",
                "multiqc/group_reports/Bpacificus/multiqc_plots/pdf/fastq_screen_plot-cnt.pdf",
                "multiqc/group_reports/Bpacificus/multiqc_plots/pdf/fastq_screen_plot-pct.pdf",
                "multiqc/group_reports/Bpacificus/multiqc_plots/pdf/fastqc-status-check-heatmap.pdf",
                "multiqc/group_reports/Bpacificus/multiqc_plots/pdf/fastqc_adapter_content_plot.pdf",
                "multiqc/group_reports/Bpacificus/multiqc_plots/pdf/fastqc_per_base_n_content_plot.pdf",
                "multiqc/group_reports/Bpacificus/multiqc_plots/pdf/fastqc_per_base_sequence_quality_plot.pdf",
                "multiqc/group_reports/Bpacificus/multiqc_plots/pdf/fastqc_per_sequence_gc_content_plot_Counts.pdf",
                "multiqc/group_reports/Bpacificus/multiqc_plots/pdf/fastqc_per_sequence_gc_content_plot_Percentages.pdf",
                "multiqc/group_reports/Bpacificus/multiqc_plots/pdf/fastqc_per_sequence_quality_scores_plot.pdf",
                "multiqc/group_reports/Bpacificus/multiqc_plots/pdf/fastqc_sequence_counts_plot-cnt.pdf",
                "multiqc/group_reports/Bpacificus/multiqc_plots/pdf/fastqc_sequence_counts_plot-pct.pdf",
                "multiqc/group_reports/Bpacificus/multiqc_plots/pdf/fastqc_sequence_duplication_levels_plot.pdf",
                "multiqc/group_reports/Bpacificus/multiqc_plots/pdf/fastqc_top_overrepresented_sequences_table.pdf",
                "multiqc/group_reports/Bpacificus/multiqc_plots/pdf/general_stats_table.pdf",
                "multiqc/group_reports/Bpacificus/multiqc_plots/png",
                "multiqc/group_reports/Bpacificus/multiqc_plots/png/SAMPLE_PAIRED_END_1_1.png",
                "multiqc/group_reports/Bpacificus/multiqc_plots/png/SAMPLE_PAIRED_END_2_2.png",
                "multiqc/group_reports/Bpacificus/multiqc_plots/png/fastq_screen_plot-cnt.png",
                "multiqc/group_reports/Bpacificus/multiqc_plots/png/fastq_screen_plot-pct.png",
                "multiqc/group_reports/Bpacificus/multiqc_plots/png/fastqc-status-check-heatmap.png",
                "multiqc/group_reports/Bpacificus/multiqc_plots/png/fastqc_adapter_content_plot.png",
                "multiqc/group_reports/Bpacificus/multiqc_plots/png/fastqc_per_base_n_content_plot.png",
                "multiqc/group_reports/Bpacificus/multiqc_plots/png/fastqc_per_base_sequence_quality_plot.png",
                "multiqc/group_reports/Bpacificus/multiqc_plots/png/fastqc_per_sequence_gc_content_plot_Counts.png",
                "multiqc/group_reports/Bpacificus/multiqc_plots/png/fastqc_per_sequence_gc_content_plot_Percentages.png",
                "multiqc/group_reports/Bpacificus/multiqc_plots/png/fastqc_per_sequence_quality_scores_plot.png",
                "multiqc/group_reports/Bpacificus/multiqc_plots/png/fastqc_sequence_counts_plot-cnt.png",
                "multiqc/group_reports/Bpacificus/multiqc_plots/png/fastqc_sequence_counts_plot-pct.png",
                "multiqc/group_reports/Bpacificus/multiqc_plots/png/fastqc_sequence_duplication_levels_plot.png",
                "multiqc/group_reports/Bpacificus/multiqc_plots/png/fastqc_top_overrepresented_sequences_table.png",
                "multiqc/group_reports/Bpacificus/multiqc_plots/png/general_stats_table.png",
                "multiqc/group_reports/Bpacificus/multiqc_plots/svg",
                "multiqc/group_reports/Bpacificus/multiqc_plots/svg/SAMPLE_PAIRED_END_1_1.svg",
                "multiqc/group_reports/Bpacificus/multiqc_plots/svg/SAMPLE_PAIRED_END_2_2.svg",
                "multiqc/group_reports/Bpacificus/multiqc_plots/svg/fastq_screen_plot-cnt.svg",
                "multiqc/group_reports/Bpacificus/multiqc_plots/svg/fastq_screen_plot-pct.svg",
                "multiqc/group_reports/Bpacificus/multiqc_plots/svg/fastqc-status-check-heatmap.svg",
                "multiqc/group_reports/Bpacificus/multiqc_plots/svg/fastqc_adapter_content_plot.svg",
                "multiqc/group_reports/Bpacificus/multiqc_plots/svg/fastqc_per_base_n_content_plot.svg",
                "multiqc/group_reports/Bpacificus/multiqc_plots/svg/fastqc_per_base_sequence_quality_plot.svg",
                "multiqc/group_reports/Bpacificus/multiqc_plots/svg/fastqc_per_sequence_gc_content_plot_Counts.svg",
                "multiqc/group_reports/Bpacificus/multiqc_plots/svg/fastqc_per_sequence_gc_content_plot_Percentages.svg",
                "multiqc/group_reports/Bpacificus/multiqc_plots/svg/fastqc_per_sequence_quality_scores_plot.svg",
                "multiqc/group_reports/Bpacificus/multiqc_plots/svg/fastqc_sequence_counts_plot-cnt.svg",
                "multiqc/group_reports/Bpacificus/multiqc_plots/svg/fastqc_sequence_counts_plot-pct.svg",
                "multiqc/group_reports/Bpacificus/multiqc_plots/svg/fastqc_sequence_duplication_levels_plot.svg",
                "multiqc/group_reports/Bpacificus/multiqc_plots/svg/fastqc_top_overrepresented_sequences_table.svg",
                "multiqc/group_reports/Bpacificus/multiqc_plots/svg/general_stats_table.svg",
                "multiqc/group_reports/Bpacificus/multiqc_report.html",
                "pipeline_info",
                "pipeline_info/nf_core_seqinspector_software_mqc_versions.yml",
                "seqfu_stats",
                "seqfu_stats/SAMPLE_PAIRED_END_1_1.tsv",
                "seqfu_stats/SAMPLE_PAIRED_END_1_1_mqc.txt",
                "seqfu_stats/SAMPLE_PAIRED_END_2_2.tsv",
                "seqfu_stats/SAMPLE_PAIRED_END_2_2_mqc.txt"
            ],
            [
                "SAMPLE_PAIRED_END_1_1_1_screen.png:md5,f52ddd0e65f23f68dd8d58d5453417db",
                "SAMPLE_PAIRED_END_1_1_1_screen.txt:md5,921b08eeb461a2aa1bd42bba53900f70",
                "SAMPLE_PAIRED_END_1_1_2_screen.png:md5,908f4fceea8aa546106efdaa7ca78f7d",
                "SAMPLE_PAIRED_END_1_1_2_screen.txt:md5,e9ea5789e356f666de27ba658704e80d",
                "SAMPLE_PAIRED_END_2_2_1_screen.png:md5,48d546767ed2a34a1eda59d97dd00c18",
                "SAMPLE_PAIRED_END_2_2_1_screen.txt:md5,a5b047f495d01bf9dcb0ae516d3c7fc3",
                "SAMPLE_PAIRED_END_2_2_2_screen.png:md5,e3684b17339d16e6fb64ace7593cc388",
                "SAMPLE_PAIRED_END_2_2_2_screen.txt:md5,fc95e5058e764596d5da182cbc0614cf",
                "fastq_screen_plot.txt:md5,642264b93871c724aaa5d38b91c752dd",
                "fastqc-status-check-heatmap.txt:md5,8e29ad6b23e03267c33bd9ed3385a44b",
                "fastqc_adapter_content_plot.txt:md5,46fbd727a4996a3f0310d0423c628445",
                "fastqc_per_base_n_content_plot.txt:md5,4eca1ef40d4d3d318b82e68e9fa3c589",
                "fastqc_per_base_sequence_quality_plot.txt:md5,218757a51c3de79a39a704c979df03da",
                "fastqc_per_sequence_gc_content_plot_Counts.txt:md5,a4035ac9f0a0c07574130758b06630d7",
                "fastqc_per_sequence_gc_content_plot_Percentages.txt:md5,637b937e023c84115ec06b529cc8ec1b",
                "fastqc_per_sequence_quality_scores_plot.txt:md5,47d60153a430e2b8da51655704c2b31b",
                "fastqc_sequence_counts_plot.txt:md5,10737092a3473fba81fbe49555512b2b",
                "fastqc_sequence_duplication_levels_plot.txt:md5,6412f57686de02e1f1c3751a7ca2be8b",
                "multiqc_SAMPLE_PAIRED_END_1_1.txt:md5,f6a5c7426b2c70dba8566b1f27d27fbf",
                "multiqc_SAMPLE_PAIRED_END_2_2.txt:md5,f531e01892d985ca20fc4924dec7b9f2",
                "multiqc_citations.txt:md5,5f52d7a0141e4234c6069df9ef575c9a",
                "multiqc_fastq_screen.txt:md5,5ef00028497d3271c9f0ea9a961db1e5",
                "multiqc_fastqc.txt:md5,bc499ad273fd1c887056c31b82544984",
                "fastq_screen_plot.txt:md5,642264b93871c724aaa5d38b91c752dd",
                "fastqc-status-check-heatmap.txt:md5,8e29ad6b23e03267c33bd9ed3385a44b",
                "fastqc_adapter_content_plot.txt:md5,46fbd727a4996a3f0310d0423c628445",
                "fastqc_per_base_n_content_plot.txt:md5,4eca1ef40d4d3d318b82e68e9fa3c589",
                "fastqc_per_base_sequence_quality_plot.txt:md5,218757a51c3de79a39a704c979df03da",
                "fastqc_per_sequence_gc_content_plot_Counts.txt:md5,a4035ac9f0a0c07574130758b06630d7",
                "fastqc_per_sequence_gc_content_plot_Percentages.txt:md5,637b937e023c84115ec06b529cc8ec1b",
                "fastqc_per_sequence_quality_scores_plot.txt:md5,47d60153a430e2b8da51655704c2b31b",
                "fastqc_sequence_counts_plot.txt:md5,10737092a3473fba81fbe49555512b2b",
                "fastqc_sequence_duplication_levels_plot.txt:md5,6412f57686de02e1f1c3751a7ca2be8b",
                "multiqc_SAMPLE_PAIRED_END_1_1.txt:md5,f6a5c7426b2c70dba8566b1f27d27fbf",
                "multiqc_SAMPLE_PAIRED_END_2_2.txt:md5,f531e01892d985ca20fc4924dec7b9f2",
                "multiqc_citations.txt:md5,5f52d7a0141e4234c6069df9ef575c9a",
                "multiqc_fastq_screen.txt:md5,5ef00028497d3271c9f0ea9a961db1e5",
                "multiqc_fastqc.txt:md5,bc499ad273fd1c887056c31b82544984",
                "SAMPLE_PAIRED_END_1_1.tsv:md5,bbd911ff3fa43212dfd4c5a5872dbdb6",
                "SAMPLE_PAIRED_END_1_1_mqc.txt:md5,cd60f746b6749cc66213d031af1b931c",
                "SAMPLE_PAIRED_END_2_2.tsv:md5,152566a128dd657da33465b4966b5799",
                "SAMPLE_PAIRED_END_2_2_mqc.txt:md5,8a3296956e094e3a95e28431e6009c40"
            ]
        ],
        "meta": {
            "nf-test": "0.9.0",
            "nextflow": "24.10.4"
        },
        "timestamp": "2025-03-25T12:00:04.338174"
    }
}